packages/headless/src/api/search/search-api-client.ts

```diff
@@ -1,4 +1,3 @@
-import AbortController from 'node-abort-controller';
```

Comment:
Related to the comment below, if we decide to remove the behaviour in all environments, we can uninstall this dependency from package.json,